frontend/src/v5/store/tickets/card/ticketsCard.selectors.ts

```diff
@@ -139,20 +142,19 @@ export const selectFilteredTickets = createSelector(
 	},
 );
 
-const selectTemplatesFilters = createSelector(
+export const selectTemplatesWithTickets = createSelector(
```

Comment:
When I select a template filter, and then try to edit it, the options I can select from only include the currently selected values.
In other words, if I select 2 templates and then I want to edit the value to include one more template, I can't


https://github.com/user-attachments/assets/0bd6d11d-61ee-4e2d-934b-55bfb61a3e5c